### ISSUE TITLE
run_ner.py training data file format

### ISSUE THREAD

**Issue by pranav-s** [huggingface/transformers#10468]
# 🚀 Feature request

Would it be possible to support text files as input files for the run_ner.py script similar to what was supported in examples/legacy/token-classification/run_ner.py ? This, I believe is the CoNLL-2003 format. The current version of the run_ner.py script in /examples/token-classification appears to support only json and csv format for training input. Could someone also tell me where I can find example files for aforementioned json and csv files ? Looking at some example files would help me format my labeled data in the required format if supporting text files is not possible.

## Motivation

I have some training data formatted as text files with one token and label per line and would like to use that as input to the run_ner.py script if possible.

**Comment by patil-suraj** [comment]
Hi @pranav-s 

We only support the `json` and `csv` files in the examples. To convert your text files to the `json` format, you could use this `datasets` script as references, which converts the `conll` text files to the `datasets` format.
https://github.com/huggingface/datasets/blob/master/datasets/conll2003/conll2003.py

**Comment by bhadreshpsavani** [comment]
Hi @pranav-s,
I found this script from `test_examples.py` file,
```
run_ner.py
  --model_name_or_path bert-base-uncased
  --train_file tests/fixtures/tests_samples/conll/sample.json
  --validation_file tests/fixtures/tests_samples/conll/sample.json
  --output_dir {tmp_dir}
  --overwrite_output_dir
  --do_train
  --do_eval
  --warmup_steps=2
  --learning_rate=2e-4
  --per_device_train_batch_size=2
  --per_device_eval_batch_size=2
  --num_train_epochs=2
```
at this path `tests/fixtures/tests_samples/conll/sample.json` you can find an example file which is having required input format

**Comment by pranav-s** [comment]
Thank you @bhadreshpsavani and @patil-suraj . This was helpful.

**Comment by lampard2a4** [comment]
Thank you @bhadreshpsavani . You saved my life!
